server/config/index.js

```diff
@@ -31,12 +41,31 @@ const getRoutingConfig = async () => {
 
   routing.defaultToNamespace =
     routing.default_to_namespace || routing.defaultToNamespace;
-  delete routing.default_to_namespace
-  
+  delete routing.default_to_namespace;
+
   return routing;
 };
 
+const getTlsConfig = () => {
+  let { tls } = readConfigSync();
```

Comment:
for reviewers:

making this `async` is possible with additional refactoring of how `temporal-client` is initialized, though imo it's not worth it as:
 - close to 0 perf benefit by making it async
 - `readConfigSync` uses cache, so the actual read is done only once
 - `temporal-client` init code is going to be less readable if refactored to async